openapi.yaml

```
@@ -7226,6 +7228,140 @@ paths:
                             "batch_description": "Nightly eval job",
                           }
                         }
+    /organization/audit_logs:
+        get:
+            summary: List user actions and configuration changes within this organization.
+            operationId: listAuditLogs
+            tags:
+                - Audit Logs
+            parameters:
+                - name: effective_at.gte
+                  in: query
+                  description: Return only events whose effective_at (Unix seconds) is greater than or equal to this value.
+                  required: false
+                  schema:
+                      type: integer
+                - name: effective_at.lte
+                  in: query
+                  description: Return only events whose effective_at (Unix seconds) is less than or equal to this value. Defaults to the current time.
+                  required: false
+                  schema:
+                      type: integer
```

note: I think you should turn this into an object as it'll result in much better SDKs, is there any reason not to?
```suggestion
                - name: effective_at
                  in: query
                  description: Return only events whose effective_at (Unix seconds) is greater than or equal to this value.
                  required: false
                  schema:
                      type: object
                      properties:
                        gte:
                          type: integer
                        lte:
                          type: integer
```

---

I did this because the documentation UI wasn't expanding the `object` type and showing `gte` and `lte`. We are actually modelling this as an object type in the API though so I will update this.